sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityComputationResults.java

```diff
@@ -90,61 +111,142 @@ public String getLogs() {
     }
 
     /**
-     * Get the sensitivity value associated with given function and given variable
+     * Get the sensitivity value associated with given function and given variable in state N.
      *
      * @param function sensitivity function
      * @param variable sensitivity variable
-     * @return the sensitivity value associated with given function and given variable
+     * @return the sensitivity value associated with given function and given variable in state N.
      */
     public SensitivityValue getSensitivityValue(SensitivityFunction function, SensitivityVariable variable) {
         Optional<SensitivityValue> returnValue = sensitivityValues.stream().filter(sensitivityValue -> sensitivityValue.getFactor().getFunction().equals(function)
-            && sensitivityValue.getFactor().getVariable().equals(variable)).findAny();
+                && sensitivityValue.getFactor().getVariable().equals(variable)).findAny();
         if (!returnValue.isPresent()) {
-            throw new NoSuchElementException(String.format("Sensitivity value not found for function %s and variable %s.", function.getId(), variable.getId()));
+            throw new NoSuchElementException(String.format(VALUE_NOT_FOUND, function.getId(), variable.getId()));
         }
         return returnValue.get();
     }
 
     /**
-     * Get the sensitivity value associated with given factor
+     * Get the sensitivity value associated with given factor in state N.
      *
      * @param factor sensitivity factor
-     * @return the sensitivity value associated with given function and given variable
+     * @return the sensitivity value associated with given function and given variable in state N.
      */
     public SensitivityValue getSensitivityValue(SensitivityFactor factor) {
         Optional<SensitivityValue> returnValue = sensitivityValues.stream().filter(sensitivityValue -> sensitivityValue.getFactor().equals(factor)).findAny();
         if (!returnValue.isPresent()) {
-            throw new NoSuchElementException(String.format("Sensitivity value not found for function %s and variable %s.", factor.getFunction().getId(), factor.getVariable().getId()));
+            throw new NoSuchElementException(String.format(VALUE_NOT_FOUND, factor.getFunction().getId(), factor.getVariable().getId()));
         }
         return returnValue.get();
     }
+
+    /**
+     * Get the status of the presence of contingencies
+     *
+     * @return true if the computation contains contingencies, false otherwise
+     */
+    public boolean contingenciesArePresent() {
+        return contingenciesPresent;
+    }
+
+    /**
+     * Get a collection of all the sensitivity values for all contingencies.
+     *
+     * @return a collection of all the sensitivity values for all contingencies.
+     */
+    public Map<String, List<SensitivityValue>> getSensitivityValuesContingencies() {
+        return sensitivityValuesContingencies;
```

Comment:
Return a copy of the map and not the map itself to prevent unsafe changes:
```java
return Collections.unmodifiableMap(sensitivityValuesContingencies);
```